README.md

```diff
@@ -1,2 +1,6 @@
 # gamebook.py
 a python library for building text based choose your own adventure games
+
+לאגם:
+עוד לא הרצתי טסטים על הקלאסים החדשים אז אין לי מושג אם הם עובדים
```

Comment:
Next time, add this comment in the PR itself.
Regarding the question. Upload some tests so we will know then 👍